rubicon_ml/client/mixin.py

```diff
@@ -23,6 +23,7 @@
 if TYPE_CHECKING:
     import dask.dataframe as dd
     import pandas as pd
+    from polars import DataFrame as polars_DataFrame
```

Comment:
nit: can we do the import consistently? either like this or with `import polars as pl` like the rest of the files